ambry-frontend/src/main/java/com.github.ambry.frontend/AmbryBlobStorageService.java

```diff
@@ -146,13 +146,13 @@ public void start() throws InstantiationException {
     getPeersHandler = new GetPeersHandler(clusterMap, securityService, frontendMetrics);
     getSignedUrlHandler =
         new GetSignedUrlHandler(urlSigningService, securityService, idConverter, accountAndContainerInjector,
-            frontendMetrics, clusterMap);
+            frontendMetrics, clusterMap, clusterName, frontendConfig);
     postBlobHandler =
         new PostBlobHandler(securityService, idConverter, idSigningService, router, accountAndContainerInjector,
```

Comment:
Unfortunately, I think there is also an issue with stitchBlob that needs to be fixed in PostBlobHandler.
```
      for (String signedChunkId : signedChunkIds) {
        signedChunkId = RequestPath.parse(signedChunkId, Collections.emptyMap(), frontendConfig.pathPrefixesToRemove, clusterName).getOperationOrBlobId(false);
```